plumgrid_playbooks/plumgrid_common.yml

```diff
@@ -20,9 +20,13 @@
     - plumgrid_common
     - create_ifcs
     - plumgrid_director
+  any_errors_fatal: yes
   vars:
+    pg_package_name: plumgrid-lxc
```

Comment:
These names are getting mixed up, all you need to define for this director role is 

pg_package_type: plumgrid-lxc

the rest it will pick up itself by looking up the pg_pkg_name dictionary. And I would recommend changing this dictionary name to pg_package only